src/tasks/installer.tasks/installer.tasks.csproj

```diff
@@ -10,15 +10,13 @@
     <PackageReference Include="NuGet.ProjectModel" Version="$(NugetProjectModelVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppToolCurrent)'">
```

Comment:
Why do you need to change the condition here? We usually prefer TFI conditions over exact TFM conditions. This make sure that this still works in case of multi-targeting for different .NETCoreApp versions.